build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-VERSION="0.4"
-MILESTONE=
+VERSION="1.1"
```

Comment:
What's versioning schema you want to use? 1.1, 1.2, etc?
For all other roles we use:

1.0.x for 4.1
1.1.x for 4.2
1.2.x for 4.3

Jut FYI, in case you want to use same schema, other that that looks OK tome.

Reply:
So the last version was 0.4 meaning developer preview. Does this mean that this release is considered stable release? If so then it would be nice to stick to our version conventions and use `1.1.0`.

Also no need to create 1.2.z unless there is some breaking change which would make the role unusable for RHV 4.2